java/code/src/com/suse/manager/reactor/utils/RhelUtils.java

```diff
@@ -156,11 +157,18 @@ public String getRelease() {
      * @return the parsed content of the release file
      */
     public static Optional<ReleaseFile> parseReleaseFile(String releaseFile) {
+        // We match here data from the system and try to find the product
+        // how it is named in SCC or sumatoolbox. This requires sometimes
+        // some changes on the string we parse.
+        //
         // AlmaLinux and AmazonLinux are also matched by the RHEL matcher
         Matcher matcher = RHEL_RELEASE_MATCHER.matcher(releaseFile);
         if (matcher.matches()) {
             String name =
                     matcher.group(1).replaceAll("(?i)linux", "").replaceAll(" ", "");
+            if (name.startsWith("Alma") || name.startsWith("Amazon")) {
```

Comment:
I wonder if we considered AlibabaLinux or we should let it for later.

Reply:
AFAIK Alibaba has a different matcher as the string is not comparable. So we should be fine for Alibaba in case we support it. Currently it is not in the toolbox/SCC.